docs/release-notes/2.42.md

```diff
@@ -0,0 +1,91 @@
+
+The design and navigation are simplified here, to focus on the essential parameters and metrics most relevant to MongoDB performance monitoring.
+
+We encourage you to explore these experimental dashboards and provide feedback in the PMM forum to help us refine them.
+
+![!image](../_images/MongoDB_Cluster_Summary.png)
+
+![!image](../_images/ReplSet_Dashboard_New.png) 
+
+### Improvements
+
+- [PMM-13075](https://perconadev.atlassian.net/browse/PMM-13075) - Added support for Ubuntu 24.04 support.
+- [PMM-12896](https://perconadev.atlassian.net/browse/PMM-PMM-12896), [PMM-12896](https://perconadev.atlassian.net/browse/PMM-PMM-12896), [PMM-12895](https://perconadev.atlassian.net/browse/PMM-12895), [PMM-12971](https://perconadev.atlassian.net/browse/PMM-12971) - Added per-database metrics collection and control of parameters, enabling more granular data gathering without overusing connections.
+- [PMM-12994](https://perconadev.atlassian.net/browse/PMM-12994) - Added labels to the `pg_replication_slot_slot_is_active` and `slot_current_wal_lsn` metrics in the postgres_exporter to identify the replication type (logical or physical) and the plugin used for the slot.
+  
+### Fixed issues
+
+- [PMM-12349](https://perconadev.atlassian.net/browse/PMM-12349) -  Fixed an issue in the ReplicaSet Summary dashboard where incorrect data was displayed when a node in the replica set was down, ensuring that the dashboard accurately reflects the status and version information of the nodes even when they are unavailable.
+
+- [PMM-12522](https://perconadev.atlassian.net/browse/PMM-12522) - Fixed a limitation in PMM Server that prevented it from monitoring more than 8 mongos instances in a large MongoDB cluster deployment.
+
+- [PMM-12880](https://perconadev.atlassian.net/browse/PMM-12880) - Fixed an issue where the `pmm-admin --tls-skip-verify` flag did not work correctly when using x509 authentication for MySQL, ensuring that the flag now properly skips certificate verification when connecting to MySQL instances with self-signed or untrusted certificates.
+
+- [PMM-12962](https://perconadev.atlassian.net/browse/PMM-12962) -  Updated the balancer metrics in MongoDB_Mongos exporter to accurately calculate and report the `mongodb_mongos_sharding_balancer_enabled` metric, providing a correct indication of whether the balancer is currently enabled or disabled in the sharded cluster.
+
+- [PMM-12989](https://perconadev.atlassian.net/browse/PMM-12989) - Fixed an issue where incorrect log entries persisted when monitoring `authentication-enabled arbiter` nodes in a MongoDB replica set, ensuring that the logs now accurately reflect the status and behavior of the arbiter nodes.
```

Comment:
I'd re-phrase to "Fixed an issue where incorrect log entries persisted when monitoring Arbiter nodes in a MongoDB replica set"